Add a variable to allow shifting the column address an arbitrary amount

U47 opened the pull request:
Found this lurking as a const in the [old C library](https://github.com/adafruit/ST7565-LCD/blob/d40d9d832be8a6ce26d6462e85de3dcf439942fc/ST7565/ST7565.cpp#L593) as (`ST7565_STARTBYTES`), and although the value presented there wasn't working for my display, I was able to get my display working correctly with a different value. Decided to make it an instance variable so it can be adjusted as needed (alongside the pagemap list, which I also needed to adjust for my display).

tekktrik commented:
Looks like `black` keeps getting angry, you can further repent to the linter using `pre-commit` :) Here's some instructions on getting started, should help you get that sorted out!

https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code

U47 commented:
> Looks like `black` keeps getting angry, you can further repent to the linter using `pre-commit` :) Here's some instructions on getting started, should help you get that sorted out!

I appreciate the help. I was trying to get away with editing directly inside GitHub since it's such a minor thing, but I've angered the testing gods. I'm still new to Adafruit's way of doing things!

I'll clone it correctly and run the hooks properly before pushing again. Thanks for that link to the info I need.

U47 committed:
Add an instance variable to allow shifting the column address an arbitrary amount

Found this lurking as a const in the [old C library][1] as (`ST7565_STARTBYTES`), and although the value presented there wasn't working for my display, I was able to get my display working correctly with a different value. Decided to make it an instance variable so it can be adjusted as needed (alongside the pagemap list, which I also needed to adjust for my display).

[1]: https://github.com/adafruit/ST7565-LCD/blob/d40d9d832be8a6ce26d6462e85de3dcf439942fc/ST7565/ST7565.cpp#L593

FoamyGuy reviewed (approved):
This change looks good to me. Prior behavior should remain as the default, but can now be more easily adjusted if user has a display that works differently. 

I don't have hardware to test, and it looks like the adafruit product for this driver is no longer available. Based on the code change I believe existing behavior should not be affected.
